### PR TITLE
[Conditions] Condition Set cleanup

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Addresses orphaned code and linting fix.

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N/A |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |

### CODE REVIEW

File: src/plugins/condition/components/ConditionSet.vue [hunk 1]
```diff
@@ -34,8 +34,10 @@
         </div>
     </section>
     <TestData :is-editing="isEditing" />
-    <ConditionCollection :is-editing="isEditing"
-                         @conditionSetResultUpdated="updateCurrentOutput" />
+    <ConditionCollection
```
Comment (JoelMcKinnon) [inline src/plugins/condition/components/ConditionSet.vue]: I like the change but had thought our standard was to put the first attribute on the first line and indent all subsequent attributes the same as the first. To fix the linting issue you would just need to move the closing bracket to its own line. Again, I like your pattern better and would agree this is more readable.

Comment (davetsay) [reply src/plugins/condition/components/ConditionSet.vue]: Yeah the only linting error is the closing bracket should be in a newline. I just prefer in this case to put all attributes on their own line. Its more natural for me to do so, and especially in this case the tag name is so long it just bothers me to indent the next line so far over.

We decided to not to enforce a strict standard, but allow the user to do what feels more comfortable to them. You can leave the first attribute on the first line if you want, but its not required. If you do that the next attribute lines up with the first attribute for indention. I believe what you cannot do is have multiple attributes in the same line.

Comment (JoelMcKinnon) [reply src/plugins/condition/components/ConditionSet.vue]: I guess I assumed that doing it the way you did would also be a lint violation. Knowing it's not, I'll adopt your pattern going forward.

File: src/plugins/condition/components/ConditionSet.vue [hunk 2]
```diff
@@ -60,11 +62,6 @@ export default {
     mounted() {
         this.conditionSetIdentifier = this.openmct.objects.makeKeyString(this.domainObject.identifier);
     },
-    beforeDestroy() {
```
Comment (JoelMcKinnon) [inline src/plugins/condition/components/ConditionSet.vue]: Why is this not needed?

Comment (shefalijoshi) [reply src/plugins/condition/components/ConditionSet.vue]: Because we're not subscribing to telemetry anymore.

**Pull request merged**